### ISSUE TITLE
Data folder being used as user config directory with default `storage.cfg`

### ISSUE THREAD

**Issue by murpii** [ddnet/ddnet#3758]
does not happen on a newly installed ddnet 15.4 version. Formatted my entire main drive with windows 10 about a week ago, meaning the current installation (steam version) I have right now is also only a week old, so I have no clue what the cause could be.

video:

https://user-images.githubusercontent.com/60477660/113776683-7575b100-972a-11eb-835e-94fdea61fc3b.mp4

clicking on the dupes does nothing

**Comment by Jupeyy** [comment]
is there a demo directory inside the data directory of the client?

**Comment by murpii** [comment]
> is there a demo directory inside the data directory of the client?

Yes, not sure why though

**Comment by Jupeyy** [comment]
changed storage.cfg maybe?

**Comment by murpii** [comment]
> changed storage.cfg maybe?

nope, never touched the storage.cfg

**Comment by Jupeyy** [comment]
Then i dunno what could cause it, maybe file permissions or stuff like that :/
Maybe we could still fix to allow both directories to work

**Comment by heinrich5991** [comment]
I think this is due to IStorage listing files/folders twice that exist in two of its search directories. This is the bug here, I think.

**Comment by def-** [comment]
IStorage does that, but I'd consider that correct. You should be able to enter each of the directories separately. At the moment inside of each of them you see different files.

**Comment by heinrich5991** [comment]
> IStorage does that, but I'd consider that correct. You should be able to enter each of the directories separately. At the moment inside of each of them you see different files.

IStorage tries to unify the different data directories. If you enter the maps/ folder, you should see all maps in the built in data directory and ~/.teeworlds/maps. If you want something fundamentally different, maybe we should remove the IStorage abstraction completely.

It seems relatively obvious to me that in the current abstraction, showing the directory only once would solve the issue: Once you enter it, you see the merged directories.

**Comment by def-** [comment]
Sure, that would also work to solve this.

**Comment by Robyt3** [comment]
Folders being shown multiple times is already covered by #5496.

So the remaining issue here is: how did the data folder end up being used as the user folder, assuming storage.cfg was never changed.
